.circleci/pages-deploy.sh

```diff
@@ -0,0 +1,18 @@
+git config user.name "$USER_NAME"
+git config user.email "$USER_EMAIL"
```

Comment:
I have these variables set as "Girder" and "girder-developers@kitware.com" respectively